src/providers/portion-provider.ts

```diff
@@ -79,61 +98,57 @@ export class PortionProvider implements IPortionProvider {
     );
     // 3. we have the portionAmountToken / portionAdjustedAmount ratio
     //    then we can estimate the portion amount for quote, i.e. what is the estimated token in amount deducted for the portion
-    //    this amount will be portionQuoteAmountToken = portionAmountToken / portionAdjustedAmount * quoteCurrencyAmount
+    //    this amount will be portionQuoteAmountToken = portionAmountToken / portionAdjustedAmount * quote
     //    CAVEAT: we prefer to use the quote currency amount OVER quote gas adjusted currency amount for the formula
     //    because the portion amount calculated from the exact out has no way to account for the gas units.
     return CurrencyAmount.fromRawAmount(
-      quoteCurrencyAmount.currency,
-      portionToPortionAdjustedAmountRatio.multiply(quoteCurrencyAmount).quotient
+      quote.currency,
+      portionToPortionAdjustedAmountRatio.multiply(quote).quotient
     );
   }
 
-  getPortionAdjustedQuote(
+  getQuote(
     tradeType: TradeType,
-    amount: CurrencyAmount,
     quote: CurrencyAmount,
-    quoteGasAdjusted: CurrencyAmount,
-    portionAmount: CurrencyAmount
+    portionQuoteAmount: CurrencyAmount
   ): CurrencyAmount {
     switch (tradeType) {
       case TradeType.EXACT_INPUT:
-        return quoteGasAdjusted.subtract(portionAmount);
+        return quote;
       case TradeType.EXACT_OUTPUT:
-        return this.getQuoteGasAndPortionAdjustedForExactOut(
-          portionAmount,
-          quote,
-          quoteGasAdjusted,
-          amount
-        );
+        return quote.subtract(portionQuoteAmount);
```

Comment:
thats because we went with the alternative approach here right? https://www.notion.so/uniswaplabs/Fees-Fields-Handshake-27f52f89b0424a7c88bb834d1757af40?d=4cce125efd73410f81ab001520df49f3#4b16f468af2347f5b1411ab49b8a5d91